### DISCUSSION TITLE
Filter clause with the operator WHERE IN

### DISCUSSION THREAD

**Discussion by pintoderian** [pocketbase/pocketbase#274] [Q&A]
Hello, first of all **congratulations** for the project.

Currently I find myself with the need to filter on a column with a specific array, for this I was thinking of using **WHERE IN**.

I'm following the documentation but I can't do the search with **WHERE IN**, I've actually achieved something similar with multiple OR's.

Is there a better way to do it?

**Comment by ganigeorgiev** [comment]
Hi, unfortunately No - there is no direct `WHERE IN/NOT IN` equivalent in the filter syntax.

Depending on how many users will want this, I may introduce a new operator(s) in the future, but for now combining multiple OR (`||`) conditions should be a good enough alternative.
